### PR TITLE
added button to open discovery drawer

### PR DESCRIPTION
**Pull request by amanewgirl**

### CODE REVIEW

File: src/app/components/toolbar/toolbar.component.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<mat-toolbar>
+    <span>{{title}}</span>
+    <button mat-raised-button> Open discovery packs</button>
+    </mat-toolbar>
```
Comment (clarke94) [inline src/app/components/toolbar/toolbar.component.html:4]: This looks like a incorrectly indented tag

Comment (clarke94) [reply src/app/components/toolbar/toolbar.component.html:4]: You should also remove the open button as it is no longer being used

Comment (amanewgirl) [reply src/app/components/toolbar/toolbar.component.html:4]: won't it be used when I add the discovery drawer?

Comment (clarke94) [reply src/app/components/toolbar/toolbar.component.html:4]: Possibly, but that will be a hamburger menu button, not a material button with text

Comment (amanewgirl) [reply src/app/components/toolbar/toolbar.component.html]: ok thats done

**Pull request merged**